pom.xml

```diff
@@ -162,14 +162,14 @@
         <repository>
             <id>splunk-artifactory</id>
             <name>Splunk Releases</name>
-            <url>http://splunk.artifactoryonline.com/splunk/ext-releases-local</url>
+            <url>https://splunk.jfrog.io/splunk/ext-releases-local</url>
```

Comment:
as per documentation, Splunk repo is now different

https://dev.splunk.com/enterprise/docs/devtools/java/logging-java/getstartedloggingjava/installloggingjava

Reply:
Also, without this fix, Travis build fails.